content/applications/essentials/activities.rst

```diff
@@ -3,236 +3,332 @@ Activities
-:guilabel:`Activity Type` from the drop-down menu.
+Activities can be scheduled from many different places within the database, from the :ref:`chatter
+<activities/chatter>` of a record, or from one of multiple views in an application, when available:
+the :ref:`Kanban view <activities/kanban>`, :ref:`list view <activities/list>`, or :ref:`activities
+view <activities/activity>`.
 
-.. tip::
-   Individual applications have a list of *Activity Types* dedicated to that application. For
-   example, to view and edit the activities available for the *CRM* application, go to
-   :menuselection:`CRM app --> Configuration --> Activity Types`.
+.. _activities/chatter:
 
-Enter a title for the activity in the :guilabel:`Summary` field, located in the :guilabel:`Schedule
-Activity` pop-up window.
+Chatter
+-------
 
-To assign the activity to a different user, select a name from the :guilabel:`Assigned to` drop-down
-menu. Otherwise, the user creating the activity is automatically assigned.
+One way that activities are created is from the *chatter* on any record.
 
-Lastly, feel free to add any additional information in the optional :guilabel:`Log a note...` field.
+Click the :guilabel:`Activities` button, located at the top of the chatter, and a
+:guilabel:`Schedule Activity` pop-up window appears. :ref:`Fill out the Schedule Activity form
+<activities/form>` to schedule a new activity.
 
-.. note::
-   The :guilabel:`Due Date` field on the :guilabel:`Schedule Activity` pop-up window auto-populates
-   based on the configuration settings for the selected :guilabel:`Activity Type`. However, this
-   date can be changed by selecting a day on the calendar in the :guilabel:`Due Date` field.
+.. image:: activities/chatter.png
+   :align: center
+   :alt: New activity type form.
 
-Lastly, click one of the following buttons:
+.. _activities/kanban:
 
-- :guilabel:`Schedule`: adds the activity to the chatter under :guilabel:`Planned activities`.
-- :guilabel:`Mark as Done`: adds the details of the activity to the chatter under :guilabel:`Today`.
-  The activity is not scheduled, it is automatically marked as completed.
-- :guilabel:`Done \& Schedule Next`: adds the task under :guilabel:`Today` marked as done, and
-  opens a new activity window.
-- :guilabel:`Discard`: discards any changes made on the pop-up window.
+Kanban view
+-----------
 
-.. image:: activities/schedule-pop-up.png
-   :alt: View of CRM leads and the option to schedule an activity.
+Another way that activities are created is from the *Kanban* view in an application.
 
-.. note::
-   Depending on the activity type, the :guilabel:`Schedule` button may be replaced by a
-   :guilabel:`Save` button, or an :guilabel:`Open Calendar` button.
+When in an application, click the :guilabel:`(Kanban)` icon in the top-right corner, and the data is
+displayed in a Kanban format. Click on the :ref:`activity icon <activities/important>` located
```

Comment:
The only reason I don't want to do that, is because it will only look like a clock if nothing is scheduled. So if a user goes in and see's a different icon, i want them to be sure of what ti is. That's why I link the 'activity icon' phrase every time to the top portion that lists how they can appear. So while I understad your logic here, I have to politely disagree. I am thinking baout users reading this, interacting in a database that has things scheduled, and don't want them to think they can't schedule osmething because the icon is wrong.